src/test/java/com/sonyericsson/rebuild/matchers/IsNotCollectionContainingStringParameterValues.java

```diff
@@ -0,0 +1,30 @@
+package com.sonyericsson.rebuild.matchers;
+
+import hudson.model.ParameterValue;
+import org.hamcrest.Description;
+import org.junit.internal.matchers.TypeSafeMatcher;
+
+import java.util.Collection;
+
+public final class IsNotCollectionContainingStringParameterValues extends TypeSafeMatcher<Collection<ParameterValue>> {
```

Comment:
I don't think you really need this class, you could just negate the result of `hasStringParamValues `